\documentclass[a4paper,12pt]{article}

% Packages
\usepackage{hyperref}
\usepackage{microtype}
\usepackage{amsmath}
\usepackage{listings}

\title{Review existing methods for Hi--C data analysis}
\author{H.L}
\date{\today}

\begin{document}
\maketitle

After a literature review, I collected the following reports 
describing methods of Hi--C data analysis:
	
\begin{enumerate}
	\item Peng, C. et al. The sequencing bias relaxed characteristics of Hi-C derived data and implications for chromatin 3D modeling. Nucleic Acids Research 41, e183–e183 (2013).

	\textbf{Pipeline used:} AutoChrom3D

	\item Zhang, Z., Li, G., Toh, K.-C. \& Sung, W.-K. 3D Chromosome Modeling with Semi-Definite Programming and Hi-C Data. Journal of Computational Biology 20, 831–-846 (2013).

	\textbf{Pipeline used:}

	\item Ay, F., Bailey, T. L. \& Noble, W. S. Statistical confidence estimation for Hi-C data reveals regulatory chromatin contacts. Genome Research (2014). doi:10.1101/gr.160374.113

	\textbf{Pipeline used:}

	\item Lu, Y., Zhou, Y. \& Tian, W. Combining Hi-C data with phylogenetic correlation to predict the target genes of distal regulatory elements in human genome. Nucleic Acids Research 41, 10391–-10402 (2013).

	\textbf{Pipeline used:}

	\item Hu, M., Deng, K., Qin, Z. \& Liu, J. S. Understanding spatial organizations of chromosomes via statistical analysis of Hi-C data. Quant Biol 1, 156–-174 (2013).

	\textbf{Pipeline used:}

	\item Imakaev, M. et al. Iterative correction of Hi-C data reveals hallmarks of chromosome organization. Nat Meth 9, 999–-1003 (2012).

	\textbf{Pipeline used:}

	\item Yaffe, E. \& Tanay, A. Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture. Nature Genetics 43, 1059–-1065 (2011).

	\textbf{Pipeline used:}

	\item Hu, M. et al. HiCNorm: removing biases in Hi-C data via Poisson regression. Bioinformatics 28, 3131–-3133 (2012).

	\textbf{Pipeline used:}

	\item Jin, F. et al. A high-resolution map of the three-dimensional chromatin interactome in human cells. Nature 503, 290–-294 (2013).

	\textbf{Pipeline used:}
\end{enumerate}

\newpage

\textbf{Questions to answer}\\

\begin{description}
	\item [Idea 1] Gene content per chromosome (in terms of active genes) and gene interactions. (You have more ~\emph{cis} than ~\emph{trans} interactions).
	\item [Idea 2] To challenge the so-called ``transcription factories''. If ``transcription
	factories'' exist, given that the vast majority of 3D chromosomal interactions are ~\emph{cis} interactions, the co-regulated genes should tend to be on the same chromosome. While this could be true in certain cases, I am pretty sure
	that there are genes on different chromosomes that are regulated by the same factor. Also mobility of chromatin in the nucleus is rather limited (see Spector and also Soutoglou publications). The ``transcription factory'' model could potentially apply in certain cases of master regulators (KLF-1 for example) (see
	Schoenfelder paper) but it is probably not the rule. 
	\item [Idea 3] Mentioned by Misteli ~\cite{Misteli:2012ea}. Hi-C data refer to populations. This may obscure the relationship of 3D chromatin structure and function. Even in the same population of cells, there are cells expressing a gene
	and others that do not (Mention the recent Spector paper on monoallelic expression here). What is the difference in the chromatin structure though when certain genes are expressed vs. when they are not? The recent development of single-cell Hi-C (if it really works as advertised)~\cite{Nagano:2013gja} may help answer this question. 
	\item [Idea 4] The ``kill many birds with one stone'' hypothesis. If we suppose that co-regulated genes (by master transcription factors) are truly in close proximity in 3D space, a favorite and very economical --in terms of changes required-- strategy in cancer would be to create master regulator behavior where it does not exist and mess up many genes all together. This could be done by fusing the activation-domain of a master regulator with the binding domain of an abundant transcription factor.
\end{description}

% Bibliography
\bibliographystyle{plain}
\bibliography{HiC}

\end{document}




